src/derivers/shared.ts

```diff
@@ -464,8 +463,11 @@ type GenerateEntropyArgs = {
  * @param args.extension - The extension bytes.
  * @returns The generated entropy bytes.
  */
-export function generateEntropy({ chainCode, extension }: GenerateEntropyArgs) {
-  return hmac(sha512, chainCode, extension);
+export async function generateEntropy({
```

Comment:
Is this exported? If so, this is now breaking.

Reply:
It's not exported.